### DISCUSSION TITLE
Install x64 application on Arm architecture

### DISCUSSION THREAD

**Discussion by azinsharaf** [chocolatey/choco#3203] [Q&A]
Hi, how can i Install a x64 application on Arm architecture? I need to install `dotnet-6.0-desktopruntime`. 
Is there any install argument in `Choco` similar to `architecture` in `Winget`?

**Comment by TheCakeIsNaOH** [comment]
@azinsharaf do you mean an amd64 application emulated on arm64? Or an arm64 native application?

If it is the first, there is a feature request at: #2172
If it is the second, there is a feature request at: #1803

**Reply by azinsharaf** [reply]
I meant to be able to install a x64 setup file on a arm based windows.
